examples/vpc.nix

```diff
@@ -0,0 +1,126 @@
+{
+  region ? "us-east-1"
+, accessKeyId 
+, ...
+}:
+with (import <nixpkgs> {}).lib;
+{
+  
+  machine =
+    {config, resources, pkgs, lib, ...}:
+    {
+      deployment.targetEnv = "ec2";
+      deployment.ec2.accessKeyId = accessKeyId;
+      deployment.ec2.associatePublicIpAddress = true;
+      deployment.ec2.region = region;
+      deployment.ec2.instanceType = "r3.xlarge";
```

Comment:
Maybe use a less beefy instance type, for those of us who start this example and then forgot they were running it :-)